README.md

```diff
@@ -15,7 +15,7 @@ file, where you want it to be replaced by the plugin.
 Currently the plugin supports the following options:
 
 - `region=eu-west-1` - to resolve that parameter in a specific region
-- `required=false` - to mark the parameter as non required. The plugin will replace the placeholder by  `""`.
+- `default=some-value` - to give a default value when the ssm parameter is optional. The plugin will throw an error when values are not defined and do not have a default.
```

Comment:
make a note on the fact that only strings are supported, I continuously fall into the trap.